cooltools/lib/numutils.py

```diff
@@ -662,15 +662,19 @@ def get_kernel(w, p, ktype):
                 ((-p<=y)&(y<=p)) )
         # kernel masked:
         kernel[mask] = 0
-    elif ktype == 'lowleft':
+    # ACHTUNG!!! UPRIGHT AND LOWLEFT ARE SWITCHED ...
```

Comment:
What I can say for sure, is that this way "locally adjusted expected"-s correlate (~1.0) with the corresponding values from hiccups ...

Reply:
I don't think I'm confusing x/y i/j - otherwise `vertical` `horizontal` would not correlate with the corresponding hiccups values ... - but they do now.

Reply:
Just leave some context here, mostly for myself:
1. I tested the `scipy.ndimage.convolve` function - was struck by the fact the kernel is inverted
2. figured out the way it was inverted in `scipy.ndimage.convolve` - and wrote scripts accordingly
3. Empirically was browsing the results  of dots calling with 4 kernels vs  just the donut alone - found something that didn't make sense (I thought) from the `lowleft` filter perspective - and flipped it ...
4. or maybe even worse - tried to use either `lowleft` or `upperright` - empirically checked results and picked one that made more sense ...
4. was calling dots "the wrong way" happily ever after with not such a huge discrepancy with hiccups - that's shocking and not, because `lowleft` is just 1 of the 4 and not always plays an important role
5. later on started properly comparing everything to hiccups and found out about  "locally adjusted expected" - not correlating with the corresponding hiccups values - only for `lowleft`
6. flipped `lowleft` 
7. now we are here ...

Should repeat the step 2! And make sure we understand that flipping properly